### DISCUSSION TITLE
Keine Kommunikation mit den HMIP Geräten

### DISCUSSION THREAD

**Discussion by ZikeDE** [OpenCCU/OpenCCU#2263] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Guten Tag,

ich habe seit mehreren Tagen Probleme mit meiner RaspberryMatic Steuerung.

Meine Geräte haben nach einer Änderung im UI immer den Status "Kommunikation gestört", ändere ich aber die Temperatur z.B. an einem Heizkörperthermostat wird der werd Teilweise an die CCU übermittelt.
Nach einem Neustart der CCU werden im UI alle Thermostate mot OFF angezeigt und in der Geräteübersicht stehen keine Werte zum Akkustand oder der Signalstärke.

Ich hatte zwischendurch einen CS von 90 ohne eine Veränderung im Haus. Die CCU steht leider im Keller (Betonwände, Jedoch nie Probleme gehabt.) Ich habe mir dann ein Pigtail + Antenne bestellt, da ich die Antenne im verdacht hatte. Daraufhin war der CS auch wieder bei 0.

Vor 3 Tagen wollte ich die RaspberryMatic dann mit der Angeschlossenen Antenne in mein Rack stellen. Nach dem Neustart war der CS dann bei 10 und die Kommunikation wieder gestört. Hab dann alles wieder zurückgebaut, jedoch ohne besserung.

Ich habe mir dann ein neues Funkmodul Bestellt und einige Geräte wieder angemeldet. Die Anmeldung funktionierte Problemlos, jedoch hat sich an der Problematik leider nichts Geändert.

Ich weiß langsam echt nicht mehr weiter...

Danke schonmal im voraus und ich hoffe wir können das Problem lösen.

### Describe the behavior you expected

[messages.txt](https://github.com/jens-maus/RaspberryMatic/files/10989269/messages.txt)

### Steps to reproduce the issue

1.
2.
3.
...

### What is the version this bug report is based on?

3.67.10.20230225

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Ich habe die Messages Log Datei oben angehangen.
```

**Comment by jens-maus** [comment]
> Ich weiß langsam echt nicht mehr weiter...

Da das kein Bug/Issue in RaspberryMatic selbst ist, sondern ein Nutzungsproblem bzw. Supportanfrage gehört sowas auch unter ein Diskussionsforum. Deshalb nun dahin verschoben und zur Diskussion freigegeben. Alternativ kannst du dich gerne natürlich an das homematic-forum.de wenden um dort nach Hilfe zu suchen.

Wenn ich mir das hier aber von dir so durchlese (CarrierSense 90%, manchmal >10%) dann wird die Ursache schon klar sein. Du hast einen Funkstörer in der näheren Umgebung des Homematic-Funkmodules.
Und das nach einem frischen Neustart die HmIP-Geräte (gerade die Batteriebetriebenen) keinen aktuellen Status/Wert zeigen ist vollkommen normal. Das System muss sich erst einpendeln bzw. die regelmäßigen Statusmitteilungen von den Sensoren erhalten. Diese können nicht aktiv abgefragt werden um die Batterien und den DutyCycle nicht zu sehr zu belasten. Einfach mal ein paar Stunden einpendeln lassen.

Insgesamt solltest du vielleicht wirklich mal einen Blick in die generellen Anfängeranleitung bzw. HomeMatic werfen. Das sollte viele deiner Fragen und Ungereimtheiten dann hoffentlich beseitigen:

https://homematic-forum.de/forum/viewtopic.php?f=1&t=22801
